lib/view_component/base.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-
+require "action_text/engine"
```

Comment:
Originally I added these lines of code to the ViewComponent I was working on in my rails application.  Then I thought it would be helpful if this functionality was available in the ViewComponent gem out of the box so that I would not have to wire things each time I needed this functionality.  As ViewComponent will be added to the next Rails release I also thought this might be something that other users would benefit from.  
That being said I will look into being how this could be added conditionally if ActionText is enabled.  Would that work?

Reply:
> I will look into being how this could be added conditionally if ActionText is enabled 

I think that's the right way to go.

> As ViewComponent will be added to the next Rails 

As noted in https://github.com/github/view_component/issues/206, this is no longer on the roadmap. _However_, we do [prioritize drop-in compatibility](https://github.com/github/view_component#design-philosophy) with Rails.

Reply:
I wonder if a way to solve the opt in would be for ViewComponent to encapsulate this logic in a Module/Concern that could be included. Either automatically if we can detect ActionText or just allow consumers to include it in their ApplicationComponent.